e2e_test/source_inline/kafka/protobuf/recover.slt

```diff
@@ -0,0 +1,97 @@
+system ok
+rpk topic create 'test-pb-struct'
+
+
+system ok
+jq -sR '{"schema":.,"schemaType":"PROTOBUF"}' << EOF | curl -X POST -H 'content-type: application/json' -d @- 'http://schemaregistry:8082/subjects/test-pb-struct-value/versions'
+syntax = "proto3";
+package test;
+message User {
+  int32 id = 1;
+  Name name = 2;
+}
+message Name {
+  string first_name = 1;
+  string last_name = 2;
+}
+EOF
+
+
+# create a source with v1 schema
+statement ok
+create source s with (
+  connector = 'kafka',
+  properties.bootstrap.server = 'message_queue:29092',
+  topic = 'test-pb-struct')
+format plain encode protobuf (
+  schema.registry = 'http://schemaregistry:8082',
```

Comment:
Use `${RISEDEV_SCHEMA_REGISTRY_URL}`, `${RISEDEV_KAFKA_BOOTSTRAP_SERVERS}` instead of hard-coded addr